### DISCUSSION TITLE
Thank you for using plain text files from external storage

### DISCUSSION THREAD

**Discussion by davidtakac** [gsantner/markor#2247] [General] [answered]
A lot of Android text editors persist their notes in app storage or a database. This results in the following problems: 

1. Users can't view their notes in another app.
2. Devs have to manually add support for backups to cloud services like Google Drive/Dropbox etc.
3. Devs have to manually add support for exporting the data so the user can back it up some other way. 

Markor solves all of these problems by just saving the notes as plain text to external storage. This is both the simplest solution and the one that gives the user the most freedom. 

And by supporting Markdown, it doubles as a fully featured note taking app with checklists, numbered lists, strikethrough, and more.

Simplicity is the hardest thing to achieve in software, which is why I truly admire Markor. Thank you for your work.

**Comment by gsantner** [comment, marked as answer]
Hello David, thanks for your feedback! You wrote many things which were the reason why Markor exists in the first place 😄 .
